pshtt.py

```diff
@@ -628,7 +660,7 @@ def is_hsts_entire_domain(domain):
 def is_hsts_preload_ready(domain):
     https = domain.https
 
-    eighteen_weeks = (https.hsts_max_age and (https.hsts_max_age >= 10886400))
+    eighteen_weeks = ((https.hsts_max_age is not None) and (https.hsts_max_age >= 10886400))
```

Comment:
This fixes a subtle bug where the value of `max-age` was getting returned as the value of `HSTS Preload Ready`, due to how Python logic short-circuiting works.